README.md

```diff
@@ -1 +1,29 @@
-# Hello-Microverse
\ No newline at end of file
+<h1>Hello Microverse</h1>
+
+<p>This is my first project with Microverse. I am creating this project to put into practice what I have learnt in my previous lessons.</p>
+
+<h2>Built With</h2>
+<p>
+<ul>
+  <li>HTML</li>
+  <li>CSS</li>
+</ul>
+</p>
+
```

Comment:
- [ ] **[OPTIONAL]:** Great job documenting your code! your README file looks very professional 🥇. However, for future projects, it would be beneficial to include the `Getting started` section from the [Microverse's README template](https://github.com/microverseinc/readme-template) 🗒️ which describes the steps other people need to follow in order to get your project set up and running locally. As this information is very important when you're open to contributions. 🤓